.github/workflows/docker-image-build-publish.yml

```diff
@@ -70,7 +71,9 @@ jobs:
           platforms: linux/amd64,linux/arm64
           push: true
           load: false
-          tags: ${{ env.IMAGE_TAG }}
```

Comment:
`IMAGE_TAG` would basically just add a tag of "master", which I don't think is useful.